client/src/components/InternshipCompanyInfo.jsx

```diff
@@ -1,57 +1,85 @@
-import { Avatar, Box, Typography } from '@mui/material';
+import { Box, Typography } from '@mui/material';
 import React from 'react';
 
+import amazonLogo from '../assets/company_logos/amazon_logo.svg';
+import appleLogo from '../assets/company_logos/apple_logo.svg';
+import genericLogo from '../assets/company_logos/generic.svg';
+import googleLogo from '../assets/company_logos/google_logo.svg';
+import metaLogo from '../assets/company_logos/meta_logo.svg';
+import microsoftLogo from '../assets/company_logos/microsoft_logo.svg';
+import netflixLogo from '../assets/company_logos/netflix_logo.svg';
+
 const InternshipCompanyInfo = (props) => {
+  const getCompanyIcon = (company) => {
+    let companyLowerCase = company.toLowerCase();
+    switch (companyLowerCase) {
+      case 'amazon':
+        return amazonLogo;
+      case 'apple':
+        return appleLogo;
+      case 'google':
+        return googleLogo;
+      case 'meta':
+        return metaLogo;
+      case 'microsoft':
+        return microsoftLogo;
+      case 'netflix':
+        return netflixLogo;
+      default:
+        return genericLogo;
+    }
+  };
+
   return (
-    <>
+    <Box
+      display="flex"
+      maxWidth={false}
+      sx={{
+        maxWidth: 'fit-content',
+      }}
+    >
       <Box
-        display="flex"
-        maxWidth={false}
+        component="img"
+        src={getCompanyIcon(props.name)}
         sx={{
-          maxWidth: 'fit-content',
+          width: '69px',
```

Comment:
Might be better to use rem instead of px => also note that this size will change since we made the job details text smaller (maybe wait for that to be merged first)

Reply:
Ok, i'll wait for that to be merged first. The 69 px was CSS from before so I just left it as is

Reply:
it has been merged :) but also it looks fine even after merge so this should be good to merge 👍